quranref/scripts/database_conn.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import BaseSettings
+#import pyArango.connection as conn
+from arango import ArangoClient
+
+class Settings(BaseSettings):
+    DB_HOST: str
+    DB_PORT: int
+    DB_NAME: str
+    DB_USERNAME: str
+    DB_PASSWORD: str
+    DB_COLLECTION: str
```

Comment:
Again collection name is  not going to change so using a variable for that doesn't make sense.